docs/guides/insar_product_guide.md

```diff
@@ -104,9 +117,9 @@ Immediately after ingesting the SLC, the state vectors are updated to use the be
 
 #### Prepare the DEM File
 
-In order to create differential InSAR products that show motion on the ground, one must subtract the topographic phase from the interferogram. The topographic phase, in this case, is replicated by using an existing DEM to calculate the actual topographic phase. This phase is then removed from the interferogram leaving just the motion or deformation signal (plus atmospheric delays and noise).  
+In order to create differential InSAR products that show motion on the ground, one must subtract the topographic phase from the interferogram. The topographic phase, in this case, is replicated by using an [existing DEM](docs/dems.md) to calculate the actual topographic phase. This phase is then removed from the interferogram leaving just the motion or deformation signal (plus atmospheric delays and noise).  
```

Comment:
"existing DEM" reads a little odd here (especially after taking out the mention that InSAR can be used to create DEMs).  Maybe "...by using a [Digital Elevation Model](docs/dems.md) (DEM) to calculate..."

Reply:
I would be inclined to leave it as-is, but I'll let @talogan be the final arbiter on that one. 
The link to the DEM file that I added is a placeholder, though; I'm not sure where that's ending up in the docs restructure, but thought it would be good to link to it wherever it ends up.

Reply:
I think that Heidi is right, but I would have said "pre-existing DEM" probably.  It is good  to reference our DEM stuff (even though I think it still needs to be updated?